### ISSUE TITLE
Put pdf link in download menu

### ISSUE THREAD

**Issue by numahell** [GeotrekCE/Geotrek-rando#768]
- Put PDF link in download menu with kml and gpx, instead using print button (or both ?)
- Add a parameter to keep base behavior if not true

**Comment by camillemonchicourt** [comment]
Yes with a parameter as it is good to give direct access with the actual print button

**Comment by camillemonchicourt** [comment]
And I think it should be one or the other, not both
